djstripe/models.py

```diff
@@ -320,6 +328,9 @@ class Customer(StripeObject):
     date_purged = models.DateTimeField(null=True, editable=False)
 
     objects = CustomerManager()
+    
+    allow_multiple_subscriptions = ALLOW_MULTIPLE_SUBSCRIPTIONS
+    retain_canceled_subscriptions = RETAIN_CANCELED_SUBSCRIPTIONS
```

Comment:
I believe it's a dj-stripe data policy to always retain cancelled subscriptions. I don't think the user should have to option to delete records this easily.

@pydanny, your thoughts?